CONTRIBUTING.md

```diff
@@ -24,19 +23,25 @@ We strive to make developing Material Components Web as frictionless as possible
 
 ### Setting up your development environment
 
-You'll need a recent version of [nodejs](https://nodejs.org/en/) to work on MDC Web. We [test our builds](https://travis-ci.com/material-components/material-components-web) using both the latest and LTS node versions, so use of one of those is recommended. You can use [nvm](https://github.com/creationix/nvm) to easily install and manage different versions of node on your system.
+You'll need a recent version of [Node.js](https://nodejs.org/) to work on MDC Web. We typically perform local development against the latest LTS (i.e. even-major-numbered) release.
 
-> **NOTE**: If you expect to commit updated or new dependencies, please ensure you are using npm 5, which will
-> also update `package-lock.json` correctly when you install or upgrade packages.
+> _NOTE_: You can use [nvm](https://github.com/creationix/nvm) to easily install and manage different versions of node on your system.
 
-Once node is installed, simply clone our repo (or your fork of it) and run `npm install`
+> _NOTE_: If you expect to commit updated or new dependencies, please ensure you are using the latest stable version of npm, which will update `package-lock.json` appropriately as well. (Run `npm i -g npm` to update.)
+
+Once Node.js is installed, simply clone our repo (or your fork of it) and install its dependencies:
 
 ```
 git clone git@github.com:material-components/material-components-web.git  # or a path to your fork
-cd material-components-web && npm i
+cd material-components-web
+npm i
 ```
 
-### Building Components
+### Building components
+
+> **NOTE**: Before building a new component, it is important to open (or comment on) an issue to confirm that the component is appropriate to be contributed to MDC Web, as the core team must then
+> share responsibility in maintaining it. Components not featured within the Material Design guidelines, or features with overly complex or specific use cases, are likely to be declined,
+> as MDC Web's goal is to provide baseline implementations of components and systems appearing in the guidelines. In these cases, you are encouraged to create your own "contrib" repository instead.
 
 Each component requires the following items in order to be complete:
 
```

Comment:
the "copy this google doc template" link...are we still happy with that template?